components/Navigation/NavigationLogin.vue

```diff
@@ -0,0 +1,34 @@
+<template>
+  <ClientOnly>
```

Comment:
minor, since the whole component is client-only why not name it NavigationLogin.client.vue instead?